src/actions/guides/http_and_routing/request_and_response.cr

```diff
@@ -169,6 +169,11 @@ class Guides::HttpAndRouting::RequestAndResponse < GuideAction
     end
     ```
 
+    Text responses are compressed automatically based on two `Lucky::Server.settings` entries. If alternate behavior is desired, these settings can be adjusted in your Lucky app in `config/server.cr`
+
+    - `Lucky::Server.settings.gzip_enabled (default false)`
```

Comment:
This is not *quite* right. Gzip is enabled by default in production https://github.com/luckyframework/lucky_cli/blob/83c69abf1db71f01161152f19ccddfe39b74c15d/src/web_app_skeleton/config/server.cr.ecr#L10. Maybe in a separate PR you could not that it is disabled in dev test but enabled in production?

Reply:
Gotcha! I was actually going by what the default was _without_ the override we manually add to `config/server.cr`, but point taken that for 99% of users reading this, they'll be using that "default"!

I'll make an update to this PR and resubmit with something like `(enabled by default in generated Lucky apps)`

Reply:
Perfect! Thanks :D